docs/setup/provision.md

```diff
@@ -0,0 +1,12 @@
+# Connect Create® 3 to Wi-Fi
+## The Basics
+Follow the main guide for getting started [here](https://edu.irobot.com/create3-setup).
```

Comment:
this gives me a 404 error. Is it expected?

Reply:
Yes, this link will not be live for a little bit.